src/pages/Admin/Room/Modal/Item/Item.tsx

```diff
@@ -115,17 +116,12 @@ const RoomModalItem: React.FC<RoomModalItemProps> = ({
         <h4 className="italic input-header">Create venue</h4>
       </S.Flex>
       <S.Flex>
-        <label id={"useUrl"} className="switch">
-          <input
-            type="checkbox"
-            id={"useUrl"}
-            name={"useUrl"}
-            checked={useUrl}
-            onChange={handleUrlToggle}
-            ref={register}
-          />
-          <span className="slider round"></span>
-        </label>
+        <Toggler
+          id="useUrl"
+          forwardedRef={register}
+          checked={useUrl}
+          onToggle={handleUrlToggle}
+        />
```

Comment:
There's no need for an `id`, because `Checkbox` nests it's `input` within the `label`, so they are inherently connected without needing the `id`.

```suggestion
        <Toggler
          name="useUrl"
          checked={useUrl}
          onChange={handleUrlToggle}
          forwardedRef={register}
        />
```

Reply:
removed `id` in 0ebf6c32892b1921edddcad9cc4e54f9d659a7a0